### PR TITLE
[I18N] *: correct bad placeholders

### PR DESCRIPTION
**Pull request by mart-e**
Detect and either remove the translation, either remove the
translation or correct it when possible

<details>
  <summary>Detected translations</summary>

```
Probable bad translation? in addons/account/i18n/de.po :
	Please check that the field 'Transfer Account' is set on the company.
	*berprüfen Sie, ob ein 'durchlaufender Posten' für die Zahlungsmethode '%s' hinterlegt wurde.
Probable bad translation? in addons/account/i18n/fr.po :
	<div style="margin: 0px; padding: 0px;">
    <p style="margin: 0px; padding: 0px; font-size: 13px;">
	<div style="margin: 0px; padding: 0px;">
    <p style="margin: 0px; padding: 0px; font-size: 13px;">
Probable bad translation? in addons/account/i18n/ml.po :
	Incorrect fiscal year date: day is out of range for month. Month: %s; Day: %s
	തെറ്റായ സാമ്പത്തിക വർഷ തീയതി: ദിവസം മാസത്തിനുള്ള പരിധിക്ക് പുറത്താണ്. മാസം: %s; ദിവസങ്ങളിൽ
Probable bad translation? in addons/account/i18n/ml.po :
	Tag name %(tag)s is used by more than one tax report line in %(country)s. Each tag name should only 
	% (രാജ്യം) s ലെ ഒന്നിൽ കൂടുതൽ നികുതി റിപ്പോർട്ട് ലൈനുകൾ% (ടാഗ്) ടാഗ് ഉപയോഗിക്കുന്നു. ഓരോ ടാഗ് നാമവും
Probable bad translation? in addons/account/i18n/ml.po :
	Tax %.2f%%
	നികുതി% .2f %%
Bad translation in addons/account/i18n/fi.po :
	Bank Statement %s
	% Tiliotteet
Bad translation in addons/account/i18n/fi.po :
	Operation not allowed. Since your statement line already received a number (%s), you cannot reconcil
	Toiminto ei ole sallittu. Koska lausuntorivisi on jo saanut numeron (% s), et voi sovittaa sitä koko
Bad translation in addons/account/i18n/fi.po :
	Recursion found for tax '%s'.
	Vero '% s' löytyi rekursiosta.
Bad translation in addons/account/i18n/fi.po :
	Transfer from %s
	Siirto kohteesta% s
Bad translation in addons/account/i18n/fi.po :
	Transfer to %s
	Siirrä kohteeseen% s
Bad translation in addons/account/i18n/fi.po :
	You cannot add/modify entries prior to and inclusive of the lock date %s. Check the company settings
	Et voi lisätä / muokata merkintöjä ennen lukituksen päivämäärää% s. Tarkista yrityksen asetukset tai
Bad translation in addons/account/i18n/fi.po :
	You have to define a sequence for %s in your company.
	Yrityksessäsi on määritettävä sekvenssi% s: lle.
Bad translation in addons/account/i18n/nl.po :
	You cannot add/modify entries prior to and inclusive of the lock date %s.
	U kan geen boekingen toevoegen/wijzigen voorafgaand en inclusief de slotdatum.
Bad translation in addons/account/i18n/ru.po :
	%s (rounding)
	(округление)
Bad translation in addons/account/i18n/tr.po :
	Corrupted data on journal entry with id %s.
	Id ile yevmiye girdisindeki bozuk veriler
Bad translation in addons/auth_signup/i18n/ro.po :
	%s connected
	conectat
Probable bad translation? in addons/base_import/i18n/pt_BR.po :
	between rows %d and %d
	entre as linhas% d e% d
Probable bad translation? in addons/base_import/i18n/ru.po :
	Could not retrieve URL: %(url)s [%(field_name)s: L%(line_number)d]: %(error)s
	Не удалось получить URL:%(url)s  [%(field_name)s : L%(line_number) d]:%(error)s 
Bad translation in addons/base_setup/i18n/fi.po :
	The following email addresses already exist: %s.
	Seuraavat sähköpostiosoitteet ovat jo olemassa:
Bad translation in addons/base_setup/i18n/fi.po :
	The following email addresses are invalid: %s.
	Seuraavat sähköpostiosoitteet ovat virheelliset
Bad translation in addons/base_setup/i18n/ru.po :
	The following email addresses already exist: %s.
	Следующие адреса электронной почты уже существуют:% s.
Bad translation in addons/base_setup/i18n/ru.po :
	The following email addresses are invalid: %s.
	Приведенные ниже электронные адреса недействительны:% s.
Bad translation in addons/hr_expense/i18n/vi.po :
	No Expense account found for the product %s (or for its category), please configure one.
	Không tìm thấy tài khoản chi phí cho sản phẩm % s (hoặc cho danh mục), vui lòng cấu hình tài khoản.
Bad translation in addons/hr_holidays/i18n/zh_CN.po :
	You cannot delete a time off which is in %s state
	您无法删除处于状态的休假
Bad translation in addons/hr_holidays/i18n/zh_TW.po :
	You cannot delete a time off which is in %s state
	您無法刪除處於狀態的休假
Bad translation in addons/l10n_cl/i18n/es.po :
	No RUT configured for partner [%i] %s
	No se configuró RUT para el cliente o proveedor [%i] %
Probable bad translation? in addons/mail/i18n/pl.po :
	You are going to discard 1 message. Do you confirm the action?
	Zamierzasz odrzucić %s wiadomość. Czy potwierdzasz działanie?
Probable bad translation? in addons/mrp/i18n/tr.po :
	You have produced %s %s of lot %s in the previous workorder. You are trying to produce %s in this on
	Önceki iş emrinde lot ürettiniz. Bu lotu burada da üretmeye çalışıyorsun.%s%s%s
Probable bad translation? in addons/mrp/i18n/zh_TW.po :
	You have produced %s %s of lot %s in the previous workorder. You are trying to produce %s in this on
	您已在前面的工作訂單中生成了 %s 批 %s。您正在嘗試在此生成 %s
Bad translation in addons/mrp/i18n/ru.po :
	You cannot have %s  as the finished product and in the Byproducts
	Вы не можете иметь% s как завершенный товар и в побочных товарам
Probable bad translation? in addons/point_of_sale/i18n/lt.po :
	Please define income account for this product: "%s" (id:%d).
	Prašome nustatyti pajamų sąskaitą šiam produktui: "%s" (ID:% d).
Bad translation rfixed addons/point_of_sale/i18n/es.po :
	%i -> %d
Bad translation in addons/point_of_sale/i18n/es.po :
	Please define income account for this product: "%s" (id:%d).
	Defina por favor la cuenta de ingresos para este producto:"%s"(id: %id).
Bad translation in addons/point_of_sale/i18n/tr.po :
	Make sure you are using IoT Box v18.12 or higher. Navigate to %s to accept the certificate of your I
	IoT Box v18.12 veya üstünü kullandığınızdan emin olun. IoT Kutunuzun sertifikasını kabul etmek için 
Probable bad translation? in addons/product/i18n/tr.po :
	%s %% discount
	%s indirim
Probable bad translation? in addons/project/i18n/nl.po :
	Prioritize Tasks by using the %s icon.%s Use the %s button to signalize to your colleagues that a ta
	Prioritiseer taken via het %s icoon. Gebruik de %s knop om uw collega's te laten weten dat een taak 
Probable bad translation? in addons/project/i18n/zh_TW.po :
	Prioritize Tasks by using the %s icon.%s Use the %s button to signalize to your colleagues that a ta
	使用 %s 圖示.%設置任務的優先順序 使用 %s 按鈕向同事發出任務已準備好下一階段的信號。
Bad translation in addons/purchase_requisition/i18n/es_CR.po :
	You have already one %s purchase order for this partner, you must cancel this purchase order to crea
	Usted tiene ya una orden de compra de% s para esta pareja, que debe cancelar esta orden de compra pa
Bad translation in addons/purchase_requisition/i18n/fi.po :
	You cannot confirm agreement '%s' because there is no product line.
	Et voi vahvistaa sopimusta '% s', koska tuotelinjaa ei ole.
Probable bad translation? in addons/sale/i18n/es.po :
	Invalid token found! Token partner %s != %s
	¡Se ha encontrado un token no válido! Socio de token != %s
Probable bad translation? in addons/sale/i18n/es_DO.po :
	Please define income account for this product: "%s" (id:%d) - or for its category: "%s".
	Por favor definir cuenta de ingreso para este producto: “%s” (id: % d) - o para su categoría: “%s”.
Bad translation in addons/sale/i18n/fi.po :
	(object.state in ('draft', 'sent') and 'Quotation - %s' % (object.name)) or 'Order - %s' % (object.n
	${object.company_id.name} ${object.state in ('draft', 'sent') and 'Tarjous' or 'Tilaus'} (Viite ${ob
Bad translation in addons/sale/i18n/fi.po :
	A journal must be specified of the acquirer %s.
	Hankkijaosapuolen% s on määritettävä päiväkirja.
Bad translation in addons/sale/i18n/fi.po :
	Invalid token found! Token acquirer %s != %s
	Virheellinen tunnus löytyi! Tokenin hankkija% s! =% S
Bad translation in addons/sale/i18n/fi.po :
	Invalid token found! Token partner %s != %s
	Virheellinen tunnus löytyi! Tunnuskumppani% s! =% S
Bad translation in addons/sale/i18n/fi.po :
	The Sales Order %s linked to the Analytic Account %s is cancelled. You cannot register an expense on
	Analyyttiseen tiliin% s liitetty myyntitilaus% s peruutetaan. Et voi rekisteröidä kulua peruutetusta
Bad translation in addons/sale/i18n/fi.po :
	The Sales Order %s linked to the Analytic Account %s is currently locked. You cannot register an exp
	Analyyttiseen tiliin% s linkitetty myyntitilaus% s on tällä hetkellä lukittu. Et voi rekisteröidä ku
Bad translation in addons/sale/i18n/fi.po :
	The Sales Order %s linked to the Analytic Account %s must be validated before registering expenses.
	Analyyttiseen tiliin% s liitetty myyntitilaus% s on vahvistettava ennen kulujen rekisteröintiä.
Bad translation in addons/sale/i18n/nl.po :
	The following products cannot be restricted to the company %s because they have already been used in
	Deze producten kunnen niet tot één bedrijf beperkt worden omdat die al gebruikt werden voor een offe
Probable bad translation? in addons/sale_stock/i18n/tr.po :
	This product is packaged by %.2f %s. You should sell %.2f %s.
	Bu ürün% .2f%s tarafından paketlenmiştir. % .2f%s satmalısınız.
Bad translation in addons/stock/i18n/pt.po :
	Invalid domain left operand %s
	Domínio inválido do operando esquerdo% ss
Bad translation in addons/stock/i18n/pt.po :
	Invalid domain right operand %s
	Domínio inválido do operando direito% ss
Bad translation in addons/stock/i18n/vi.po :
	<br>A need is created in <b>%s</b> and a rule will be triggered to fulfill it.
	<br> Một nhu cầu được tạo ra trong <b>% s </ b> và một quy tắc sẽ được kích hoạt để đáp ứng nhu cầu 
Probable bad translation? in addons/web/i18n/hi.po :
	Loading (%d)
	लोड हो रहा है (% d)
Bad translation in addons/web/i18n/es_CO.po :
	No value found for the field %s for value %s
	No hay valores encontrados para el campo para valor
Bad translation in addons/web/i18n/fi.po :
	Add %s
	Lisää
Probable bad translation? in addons/web/i18n/fi.po :
	Lightness %
	Kirkkaus %s
Probable bad translation? in addons/calendar/i18n/mn.po :
	Meeting '%s' starts '%s' and ends '%s'
	Уулзалт '%s' -д эхэлж '%s'-д дуусна
Probable bad translation? in addons/gamification/i18n/de.po :
	<br/> %(rank)d. %(user_name)s - %(reward_name)s
	<br/>
Probable bad translation? in addons/gamification/i18n/es.po :
	<br/> %(rank)d. %(user_name)s - %(reward_name)s
	%(reward_name)s
Probable bad translation? in addons/gamification/i18n/fr.po :
	<br/> %(rank)d. %(user_name)s - %(reward_name)s
	<br/>
Probable bad translation? in addons/payment/i18n/ro.po :
	The transaction %s with %s for %s has been cancelled with the following message: %s
	Tranzacția %scu %s  pentru%s a fost anulată cu următorul mesaj:
Probable bad translation? in addons/payment/i18n/vi.po :
	The transaction %s with %s for %s has been confirmed. The related payment is posted: %s
	Giao dịch %s với %s cho %s đã được xác nhận. Thanh toán liên quan là: %s đã vào sổ: %s
Probable bad translation? in addons/payment/i18n/zh_TW.po :
	The transaction %s with %s for %s has been confirmed. The related payment is posted: %s
	%s的%s交易%s已獲得確認。等待獲取發送付款狀態......
Bad translation in addons/payment/i18n/fi.po :
	A journal must be specified of the acquirer %s.
	Hankkijaosapuolen% s on määritettävä päiväkirja.
Bad translation in addons/payment/i18n/fi.po :
	Invalid token found! Token acquirer %s != %s
	Virheellinen tunnus löytyi! Tokenin hankkija% s! =% S
Bad translation in addons/payment/i18n/fi.po :
	Invalid token found! Token partner %s != %s
	Virheellinen tunnus löytyi! Tunnuskumppani% s! =% S
Bad translation in addons/payment_authorize/i18n/vi.po :
	Authorize: received data with missing reference (%s) or trans_id (%s) or fingerprint (%s)
	Ủy quyền: dữ liệu nhận được với tham chiếu bị thiếu () hoặc trans_id () hoặc dấu vân tay ()
Probable bad translation? in addons/website_blog/i18n/es_CO.po :
	has been published on the
	Se ha publicado un nuevo artículo %s en el blog %s.
Probable bad translation? in addons/website_blog/i18n/es_EC.po :
	has been published on the
	Se ha publicado una nueva entrada llamada %s en el blog %s.
Probable bad translation? in addons/website_blog/i18n/kab.po :
	has been published on the
	Asentel n wawal amaynut %s iţusegded ɣef  ublug %s. 
Probable bad translation? in addons/website_event_sale/i18n/sv.po :
	Sorry, The %(ticket)s tickets for the %(event)s event are sold out.
	Tyvärr, alla %(ticket) biljetter för %(event) är utsålda.
Probable bad translation? in addons/website_forum/i18n/hu.po :
	65% more chance to get an
        answer
	65%-al több lehetőség, hogy
        megválaszolják
Probable bad translation? in addons/website_forum/i18n/zh_CN.po :
	45% of questions shared
	45%% 的问题被共享
Bad translation in addons/website_slides/i18n/zh_CN.po :
	Attendees of %s
	与会者的
Bad translation in addons/website_slides/i18n/zh_CN.po :
	The following recipients have no user account: %s. You should create user accounts for them or allow
	以下收件人没有用户帐户：。 您应该为它们创建用户帐户或允许在配置中进行外部注册。
Bad translation in addons/website_slides/i18n/zh_TW.po :
	Attendees of %s
	與會者的
Probable bad translation? in addons/website_slides/i18n/zh_TW.po :
	Complete a course
	教師 %s 在課程 %s 中標出單元完成 %s
Bad translation in addons/website_slides/i18n/zh_TW.po :
	Create new %s '%s'
	創建新的
Bad translation in addons/website_slides/i18n/zh_TW.po :
	The following recipients have no user account: %s. You should create user accounts for them or allow
	以下收件人沒有使用者帳戶：。 您應該為它們創建使用者帳戶或允許在配置中進行外部註冊。
Bad translation in addons/crm_livechat/i18n/mn.po :
	Created a new lead: <a href="#" data-oe-id="%s" data-oe-model="crm.lead">%s</a>
	Шинэ сэжих үүссэн
Bad translation in addons/mrp_account/i18n/ro.po :
	The inventory valuation of some products %s is automated. You can only update their cost from the pr
	Evaluarea stocului unor produse este automatizată. Puteți actualiza costurile lor numai din formular
Bad translation in addons/mrp_account/i18n/ru.po :
	The inventory valuation of some products %s is automated. You can only update their cost from the pr
	Складскую оценку некоторых товаров% s автоматизировано. Вы можете обновить их стоимость только из фо
Probable bad translation? in addons/payment_payulatam/i18n/nl.po :
	PayU Latam: received data with missing reference (%s) or transaction id (%s) or sign (%s)
	PayU Latam: data ontvangen met ontbrekende referentie () of transactie id () of teken (%s)
Bad translation in addons/repair/i18n/pt_BR.po :
	Please define an accounting sales journal for the company %s (%s).
	Por favor defina um diário contábil de vendas para essa companhia.
Bad translation in addons/sale_coupon/i18n/ru.po :
	%s %s discount on total amount
	% S% s скидка на общую сумму
Bad translation in addons/sale_coupon/i18n/ru.po :
	%s%% discount on %s
	% S %% скидка на% s
Bad translation in addons/sale_coupon/i18n/ru.po :
	%s%% discount on cheapest product
	% S %% скидка на самый дешевый товар
Bad translation in addons/sale_coupon/i18n/ru.po :
	%s%% discount on total amount
	% S %% скидка на общую сумму
Bad translation in addons/sale_coupon/i18n/ru.po :
	A minimum of %s %s should be purchased to get the reward
	Минимум% s% sпотрибно приобрести, чтобы получить награду
Bad translation in addons/sale_coupon/i18n/ru.po :
	Free Product - %s
	Бесплатный продукт -% s
Bad translation in addons/sale_coupon/i18n/ru.po :
	Promo code %s has been expired.
	Срок действия промо-кода% s закончился.
Bad translation in addons/sale_coupon/i18n/ru.po :
	The code %s is invalid
	Код% s недействителен
Bad translation in addons/sale_coupon/i18n/ru.po :
	The coupon program for %s is in draft or closed state
	Купонная программа% Sу состоянии черновика или в закрытом состоянии
Bad translation in addons/sale_coupon/i18n/ru.po :
	This coupon %s exists but the origin sales order is not validated yet.
	Этот купон% s существует, но оригинальное заказ на продажу еще не подтверждено.
Bad translation in addons/sale_coupon/i18n/ru.po :
	This coupon %s has been used or is expired.
	Этот купон% s был использован или закончился.
Bad translation in addons/uom/i18n/fi.po :
	The unit of measure %s defined on the order line doesn't belong to the same category than the unit o
	Tilausrivillä määritetty mittayksikkö% s ei kuulu samaan kategoriaan kuin tuotteessa määritetty mitt
Bad translation in addons/uom/i18n/fi.po :
	UoM category %s should have a reference unit of measure. If you just created a new category, please 
	UoM-luokassa% s tulisi olla vertailumittayksikkö. Jos olet juuri luonut uuden kategorian, tallenna e
Bad translation in addons/uom/i18n/fi.po :
	UoM category %s should only have one reference unit of measure.
	UoM-luokassa% s tulisi olla vain yksi mittayksikkö.
Bad translation in addons/uom/i18n/sk.po :
	UoM category %s should have a reference unit of measure. If you just created a new category, please 
	Kategória mernej jednotky môže mať len jednu referenčnú mernú jednotku, zaznamenaj ju najprv prosím.
Bad translation in addons/account_bank_statement_import/i18n/es.po :
	This file doesn't contain any transaction for account %s.
	Este archivo no contiene ninguna transacción para la cuenta
Probable bad translation? in odoo/addons/base/i18n/de.po :
	The server refused the sender address (%(email_from)s) with error %(repl)s
	Der Server hat die Absenderadresse abgelehnt (%(email_from)s) mit dem Fehler
Probable bad translation? in odoo/addons/base/i18n/it.po :
	Field %r used in attributes must be present in view but is missing:
	Campo % usato negli attributi mancante, è obbligatorio nella vista:
Probable bad translation? in odoo/addons/base/i18n/pt_BR.po :
	EU Mini One Stop Shop (MOSS)
	Extensão não reconhecida: deve ser uma destas .csv, .po, or .tgz (received .%s).
Probable bad translation? in odoo/addons/base/i18n/ru.po :
	%S - Seconds [00,61]."
	 %s  — Секунды [00—61]."
Probable bad translation? in odoo/addons/base/i18n/ru.po :
	5. %H:%M:%S      ==&gt; 18:25:20
	5. %H:%M:%s       → 18:25:20
Probable bad translation? in odoo/addons/base/i18n/ru.po :
	6. %I:%M:%S %p  ==&gt; 06:25:20 PM
	6. %I:%M:%s  %p  → 06:25:20 PM
Probable bad translation? in odoo/addons/base/i18n/ru.po :
	8. %S              ==&gt; 20
	8. %s               → 20
Probable bad translation? in odoo/addons/base/i18n/tr.po :
	Empty dependency in %r
	% r'de boş bağımlılık
Probable bad translation? in odoo/addons/base/i18n/tr.po :
	Field %r used in attributes must be present in view but is missing:
	Niteliklerde kullanılan % r alanı görünmelidir, ancak mevcut değil:
Probable bad translation? in odoo/addons/base/i18n/vi.po :
	Invalid aggregation function %r.
	Hàm tổng hợp không hợp lệ% r.
Probable bad translation? in odoo/addons/base/i18n/vi.po :
	Invalid field specification %r.
	Đặc tả trường không hợp lệ% r.
Probable bad translation? in odoo/addons/base/i18n/zh_CN.po :
	The table %r if used for other, possibly incompatible fields.
	如果此表% r 用于其他可能不兼容的字段.
Bad translation in odoo/addons/base/i18n/bg.po :
	Model %s does not exist!
	Модел с % не съществува!
Bad translation in odoo/addons/base/i18n/ca.po :
	Unable to process module "%s" because an external dependency is not met: %s
	Impossible processar el mòdul "% s" perquè no es va resoldre la dependència externa:% s
Probable bad translation? in odoo/addons/base/i18n/ca.po :
	Unable to upgrade module "%s" because an external dependency is not met: %s
	Impossible actualitzar el mòdul "%s" perquè hi ha una dependència externa no resolta:% s
Bad translation in odoo/addons/base/i18n/mn.po :
	You are trying to install incompatible modules in category "%s":
	Та энэ ангиллын хувьд тохирохгүй модуль суулгахыг оролдож байна.
```
</details>

"Bad translation in" are for files were the error was corrected (or translation remove)
"Probablem bad translation?" have not been modified

Detection/correction made with https://github.com/mart-e/odoo-scripts/blob/master/transifex/pull-translations.py#L21

### CODE REVIEW

File: addons/account_bank_statement_import/i18n/es.po [hunk 1]
```diff
@@ -754,7 +754,7 @@ msgstr "Este archivo no contiene ningún estado de cuenta %s."
 #: code:addons/account_bank_statement_import/account_bank_statement_import.py:0
 #, python-format
 msgid "This file doesn't contain any transaction for account %s."
-msgstr "Este archivo no contiene ninguna transacción para la cuenta"
+msgstr ""
```
Comment (pedrobaeza) [inline addons/account_bank_statement_import/i18n/es.po:757]:
```suggestion
msgstr "Este archivo no contiene ninguna transacción para la cuenta %s."
```

Comment (mart-e) [reply addons/account_bank_statement_import/i18n/es.po:757]: @pedrobaeza this isn't a PR to be merged, just to see the problem and see what we can do about. Internal discussion about integrating this kind of test in the runbot/synchronisation bot

**Comment by nim-odoo** [comment]
That would be great to integrate such in the standard process! However I don't know how that could be interfaced with Transifex to automatically remove the incorrect translations on both sides.

File: addons/account_bank_statement_import/i18n/es.po (hunk 1 above)
Comment (pedrobaeza) [reply addons/account_bank_statement_import/i18n/es.po:757]: Ah, OK, thanks for the clarification.

**Comment by pedrobaeza** [comment]
I still think that this should be handled at other level: ORM translation part, as the source of incorrect translations can be other (Weblate for OCA, DB ir.translation direct manipulation, custom modules translations...). For me, ideally when you call `_()` with a source text with placeholders, then it should be checked if the translated part has the same number of placeholders, and if not, return the source term instead. This way, we avoid ugly errors on UI that has nothing to do with the business logic. For example, all my customers were block 2 weeks ago without access to the sequence form view due to a placeholder error in this term:

https://www.transifex.com/odoo/odoo-12/translate/#es/base/151498632

I know this exact case is what you are trying to prevent other way, but think that a same level problem can come from the other mentioned sources.

**Comment by nim-odoo** [comment]
@pedrobaeza I think both approaches are complementary, not exclusive. If we hide the incorrect translations under the carpet, it's likely that they will never be corrected (and people will ask why some translated term doesn't show up). Therefore, correcting/removing the incorrect translations is necessary.

But I agree that an incorrect translation must never block a business process. I'm not sure of what should be the best solution: it's not only a matter of the number of placeholders, see for example https://github.com/odoo/enterprise/pull/10395. `and/or` were translated, while they shouldn't have been.

**Comment by pedrobaeza** [comment]
Yeah, I know there's not a correct balance and you are trying to do your best.

About the possible failure reasons, indeed there's not only placeholders, but the most usual things can be handled (and expanded in future if more cases are detected):

- Incorrect number of placeholders
- Python expressions
- Jinja expressions
- Malformed HTML
- ...

Another possible complementary action: on ir.translation record, add a computed field that indicates that the translation is not correct according these rules, and show it through color, icon or similar on the translated terms tree view.

**Comment by mart-e** [comment]
> Another possible complementary action: on ir.translation record, add a computed field that indicates that the translation is not correct according these rules, and show it through color, icon or similar on the translated terms tree view.

@pedrobaeza you mean #33220 ? ;-)

**Comment by pedrobaeza** [comment]
I see you have caught my concept :stuck_out_tongue_closed_eyes:

**Comment by mart-e** [comment]
Corrected the detected ones on Transifex. To be decided if we are setting the filter on the synchronisation or different places

**Pull request closed without merging**